t/native_cp_al/native_cp_al.t.in

```diff
@@ -0,0 +1,39 @@
+#!@PERL@
+use strict;
+use Test::More tests => 11; # pack here your amount of subtests
+use SysWrap;
+
+#
+# Test native_cp_al works with cmd_cp option not set and sync_first on (see contig).
+#
+
+ok(! remove_snapshot_root(),
+	" snapshot root does not exist before testing starts");
+
+ok(0 == rsnapshot("-c @TEST@/native_cp_al/conf/native_cp_al.conf sync 2>&1"),
+	" sync failed");
```

Comment:
An exit code of 0 means "success", note failure!

Reply:
:+1: